verified-reviews/utils/client.ts

```diff
@@ -0,0 +1,180 @@
+import { fetchAPI } from "../../utils/fetch.ts";
+import { Ratings, Reviews, VerifiedReviewsFullReview } from "./types.ts";
+import { Product } from "../../commerce/types.ts";
+import { ConfigVerifiedReviews } from "../mod.ts";
+import { context } from "deco/live.ts";
+
+export type ClientVerifiedReviews = ReturnType<typeof createClient>;
+
+export interface PaginationOptions {
+	count?: number;
+	offset?: number;
+	order?:
+		| "date_desc"
+		| "date_ASC"
+		| "rate_DESC"
+		| "rate_ASC"
+		| "helpfulrating_DESC";
+}
+
+const messageError = {
```

Comment:
this seems more suitable
```suggestion
const MessageError = {
```

Reply:
but is a nit (you don't need to fix, if you don't want to)